### DISCUSSION TITLE
Exposed ports using Caddy

### DISCUSSION THREAD

**Discussion by techserver5110** [nextcloud/all-in-one#879] [Questions] [answered]
Hi all,

I've set up Nextcloud AIO using Caddy and when I run nmap I can see that among the expected open ports I see "8080/tcp open  http-proxy" 

Should the 8080 port be exposed publicly in this way or do I need to configure something to make it internal?

All the best!

Paul

**Comment by Zoey2936** [comment, marked as answer]
https://github.com/nextcloud/all-in-one#explanation-of-used-ports

**Reply by Zoey2936** [reply]
How do you access your aio interface? Over a reverse proxy or over port 8443/8080? And if over reverse proxy, do you proxy to port 8080 or 8443?

**Reply by techserver5110** [reply]
I'm using reverse proxy Caddy and access the AIO interface via ipaddress :8080

Docker compose is similar to:
`version: "3.8"

volumes:
  nextcloud_aio_mastercontainer:
    name: nextcloud_aio_mastercontainer

services:
  nextcloud:
    image: nextcloud/all-in-one:latest
    restart: always
    container_name: nextcloud-aio-mastercontainer
    volumes:
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
      - /var/run/docker.sock:/var/run/docker.sock:ro
    ports:
      - 8080:8080
    environment: # Is needed when using any of the options below
      - APACHE_PORT=11000 

  caddy:
    image: caddy:alpine
    restart: always
    container_name: caddy
    volumes:
      - ./Caddyfile:/etc/caddy/Caddyfile
      - ./certs:/certs
      - ./config:/config
      - ./data:/data
      - ./sites:/srv
    network_mode: "host"`

**Reply by Zoey2936** [reply]
Then I don't understand the question, if you open the aio interface using port 8080, then it should be normal that port 8080 is open?

**Reply by techserver5110** [reply]
Thanks for the help - you are of course right - my mistake.
